### ISSUE TITLE
[469d] Spotlight lightcone display

### ISSUE THREAD

**Issue by Nathkel** [OldUnreal/UnrealTournamentPatches#1608]
In order to take the guesswork out of setting up directional lights, I sure would love to see the angle of effect when setting the light cone property. Perhaps a sector can be drawn that takes into the account the light cone width in relation to the light radius.

**Comment by Nathkel** [comment]
Yes thats what i do now. But its still guessing, and i have to adjust with realtime view when i need a precise spotlight. A sector outline would show me in top or side view exactly where the light is going.

**Comment by SeriousBuggie** [comment]
I guess you mean something like in UT3
![img](https://github.com/user-attachments/assets/b9c327f9-29cb-45d2-b65e-2cea4701e35f)

**Comment by Nathkel** [comment]
I havent used UT3, so I guess. 
I myself was more interested in the orthographic views because that's how I work setting up maps.
